custom/opm/reports.py

```diff
@@ -914,10 +919,6 @@ def redis_key(self):
         return redis_key
 
     @property
-    def total_records(self):
-        return len(super(MetReport, self).rows)
```

Comment:
`rows` isn't memoized, so this was causing the calculation for the report to be run twice.